test/libsolidity/smtCheckerTests/imports/simple_imported_fail_no_pragma.sol

```diff
@@ -6,7 +6,9 @@ contract A {
 }
 ==== Source: B.sol ====
 import "A.sol";
-pragma experimental SMTChecker;
 contract C is A {}
+// ====
+// SMTEngine: all
 // ----
-// Warning 6328: (A.sol:49-62): CHC: Assertion violation happens here.\nCounterexample:\n\nx = 0\n\nTransaction trace:\nC.constructor()\nA.f(0)
+// Warning 6328: (A.sol:49-62): CHC: Assertion violation happens here.\nCounterexample:\n\nx = 0\n\nTransaction trace:\nA.constructor()\nA.f(0)
+// Warning 6328: (A.sol:49-62): CHC: Assertion violation happens here.\nCounterexample:\n\nx = 0\n\nTransaction trace:\nA.constructor()\nA.f(0)
```

Comment:
Is this test still relevant? Judging by the name, it was checking what happens when only one file has the pragma but now it's always enabled for all files, right?